Expensify/App
# feat: add image loading placeholder

### Details

Add a loading indicator while loading an image.
### Fixed Issues

$ https://github.com/Expensify/App/issues/7584

### Tests

1. Open a chat with images
2. Verify that a loading indicator appears for images that are loading

### QA Steps

1. Same as test steps

### Tested On


### Screenshots


#### Web

<img width="1440" alt="web" src="https://user-images.githubusercontent.com/77761491/153769679-2124bc32-3439-489b-a6c8-a462db07d98f.png">

#### Mobile Web

![mweb](https://user-images.githubusercontent.com/77761491/153769691-7492b051-2324-4da6-804e-1a9e4336a839.png)

#### Desktop

<img width="1440" alt="desktop" src="https://user-images.githubusercontent.com/77761491/153770252-7a0ea406-c7ec-4662-b1d4-136c8d33d06e.png">

#### iOS

![ios](https://user-images.githubusercontent.com/77761491/153769702-274a98c8-9287-4efe-9859-7da97510c959.png)

#### Android

![android](https://user-images.githubusercontent.com/77761491/153769714-ee0d2beb-94d7-49c6-a080-275efb4f84d2.png)
